amivapi/media.py

```diff
@@ -10,6 +10,7 @@
 
 """
 
+from bson import tz_util
```

Comment:
Why are we importing the tz from bson? This seems a odd place to take timezone information from.